resources/profiles/Prusa/process/process_common_miniis.json

```diff
@@ -0,0 +1,55 @@
+{
+  "type": "process",
+  "name": "process_common_miniis",
+  "from": "system",
+  "instantiation": "false",
+  "inherits": "fdm_process_common",
+  "initial_layer_speed": "30",
+  "initial_layer_infill_speed": "80",
+  "outer_wall_speed": "45",
+  "inner_wall_speed": "80",
+  "sparse_infill_speed": "115",
+  "internal_solid_infill_speed": "140",
+  "top_surface_speed": "80",
+  "gap_infill_speed": "60",
+  "travel_speed": "400",
+  "bridge_speed": "35",
+  "internal_bridge_speed": "50",
+  "small_perimeter_speed": "45",
+  "travel_jerk": "8",
+  "outer_wall_jerk": "7",
+  "inner_wall_jerk": "8",
+  "default_jerk": "8",
+  "infill_jerk": "8",
+  "top_surface_jerk": "7",
+  "initial_layer_jerk": "7",
+  "default_acceleration": "2000",
+  "initial_layer_acceleration": "500",
+  "top_surface_acceleration": "1000",
+  "travel_acceleration": "4000",
+  "sparse_infill_acceleration": "4000",
+  "internal_solid_infill_acceleration": "3000",
+  "inner_wall_acceleration": "2000",
+  "outer_wall_acceleration": "1500",
+  "bridge_acceleration": "1500",
+  "exclude_object": "1",
+  "overhang_1_4_speed": "80%",
+  "overhang_2_4_speed": "25",
+  "overhang_3_4_speed": "20",
+  "overhang_4_4_speed": "15",
+  "sparse_infill_pattern": "cubic",
+  "top_shell_thickness": "0.7",
+  "top_shell_layers": "5",
+  "bottom_shell_thickness": "0.5",
+  "bottom_shell_layers": "4",
+  "elefant_foot_compensation": "0.2",
+  "slowdown_for_curled_perimeters": "1",
+  "infill_anchor_max": "12",
+  "sparse_infill_anchor": "2,5",
+  "infill_wall_overlap": "10%",
+  "enable_arc_fitting": "1",
+  "support_speed": "100",
+  "precise_outer_wall": "1",
+  "overhang_reverse": "1",
+  "wall_sequence": "inner-outer-inner wall"
```

Comment:
I don't recommend set default to sandwich mode.
It will fail models with steep overhangs.
For default settings, "safer" option is preferred.

Reply:
My tests show that the bow of a benchy gets printed flawless with the sandwich mode active. With normal inner outer setting the bow gets ugly dents especially on bigger nozzles and the speed profiles. So I thought it would make a good default setting. 
![IMG_5708](https://github.com/SoftFever/OrcaSlicer/assets/36260775/25bcf99a-4bb1-45cd-a4f7-d064363aba57)
![IMG_5707](https://github.com/SoftFever/OrcaSlicer/assets/36260775/f2308667-9d8e-417a-8e20-9fefbf22ae55)

Reply:
Thanks for your feedback an reasoning. I got to play a little bit with the cooling settings and took a look at the PrusaSlicer settings and found out, that the layer_time settings which I copied from the Mini were too low. I will close this pull request and make another, when I finished the filament settings.